connector/connector.py

```diff
@@ -422,6 +422,9 @@ def to_openerp(self, external_id, unwrap=False):
             bindings = getattr(bindings, self._openerp_field)
         return bindings
 
+    # 2016-10-25: compatible with old implementations
+    to_openerp = to_odoo
```

Comment:
@pedrobaeza My main concern is to not break the existing API. I've cases where we have override this method to add additional checks and implement a more complex mapping rule. I don't understand the motivation behind this change on V8 and IMO we must avoid this kind of change on a stable release.
:-1: